### PR TITLE
Removes passive income from the Security Budget 2 [Electric Boogaloo]

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

There is no good way to balance around their budget. Due to the Economy subsystem not respecting the universal laws of time it will be permanently out of wack. This is better than playing whack-a-mole with 100k sec budgets 30 minutes into the game.

They now start with 10k, you can deposit and withdraw but no money is added on the Economy tick

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Sec budget gets 10k roundstart and no more

### What general grouping does this PR fall under? 
argo rebalancing

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

# Changelog

"why did you open this pr tiwce ?+!"?+??" -alexkar598
Because I flipped directions and that taints the upvotes/downvotes

:cl:  
rscdel: The Security budget now only gets a flat sum of money each shift.
/:cl:

### CODE REVIEW

**Review by alexkar598 — COMMENTED** [review]
Half measures.

Only way to bring back cargo to even a shell of what it once was(not much), you need to remove it completly.

10k is still enough to bring a shuttle load of i win implanters and send it back with 5 stacks of plasma

**Comment by Bibby0110** [comment]
> Half measures.
> 
> Only way to bring back cargo to even a shell of what it once was(not much), you need to remove it completly.
> 
> 10k is still enough to bring a shuttle load of i win implanters and send it back with 5 stacks of plasma

If you’ll approve a full removal I’ll reopen my old PR

**Comment by alexkar598** [comment]
I wish I could, maintainers arent allowed to merge big changes or something now

**Comment by Bibby0110** [comment]
> I wish I could, maintainers arent allowed to merge big changes or something now

This whole budget debacle has been half-measures all the way haha

The initial retention of it was to appease people, the later nerfs instead of removing it, and now this. Such goodness

**Comment by MegaEmpirical** [comment]
So how does Sec get more budget then?

**Comment by Bibby0110** [comment]
> So how does Sec get more budget then?

They don’t, that’s the point

**Comment by MegaEmpirical** [comment]
You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?

**Comment by Bibby0110** [comment]
> You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?

Yep, this is the alternative to just removing the budget card which is the imo better option

**Comment by Bibby0110** [comment]
> You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?

Your downvotes mean nothing if you propose no alternative, simply just so negative...

**Comment by MegaEmpirical** [comment]
> > You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?
> 
> Your downvotes mean nothing if you propose no alternative, simply just so negative...

It doesn't matter, you've stated the point is to make it so that the sec budget cannot gain money. This immediately makes our desires counter so it doesn't matter what I suggest as our goals are different.

**Comment by Bibby0110** [comment]
> > > You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?
> > 
> > 
> > Your downvotes mean nothing if you propose no alternative, simply just so negative...
> 
> It doesn't matter, you've stated the point is to make it so that the sec budget cannot gain money. This immediately makes our desires counter so it doesn't matter what I suggest as our goals are different.

Then say what your goal is lol?

**Comment by MegaEmpirical** [comment]
For Sec to be able to maintain and/or grow their budget so that officers can continue to receive a reasonable paycheck.

**Comment by Bibby0110** [comment]
> For Sec to be able to maintain and/or grow their budget so that officers can continue to receive a reasonable paycheck.

Paychecks are flat, not based on how much is in the budget.

The budget card is barely used outside of nukies round so we will observe a standard 90 minutes round.

To expend the whole budget on salaries you'd have to have the following amount of security:
1 HoS
1 Warden
1 Detective
8 officers.
This doesn't happen very often, and if it does the round is very much not going to be 90 minutes long. Consider a more realistic 60 minute round at higher pops and we get the following sec needed to expend the budget:
1 HoS, 1 warden, 1 detective, 15 officers.
Consider you spend 40% of the budget on Corgis or something:
1 HoS, 1 warden, 1 detective, 7 officers.

"What about nukie rounds then?"
You'll be dead or the round is over before you notice you're not getting paid.

**Comment by ghost** [comment]
> 
> 
> > You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?
> 
> Your downvotes mean nothing if you propose no alternative, simply just so negative...

his alternative is dont merge it you dent

**Comment by Bibby0110** [comment]
> > > You've already made paychecks be proportional to the budget and now you want officers to be unable to earn money?
> > 
> > 
> > Your downvotes mean nothing if you propose no alternative, simply just so negative...
> 
> his alternative is dont merge it you dent

His alternative is leaving something broken in the game*

Fixed that for you, thank me later

**Comment by JamieD1** [comment]
This is just a mess, will need to revisit.

**Comment by Bibby0110** [comment]
> This is just a mess, will need to revisit.

When are you going to do that exactly

**Review by JamieD1 — APPROVED** [review]
